core-database/build.gradle.kts

```
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+@kotlin.Suppress("DSL_SCOPE_VIOLATION")
```

Small nit: On my side, it looks like the `kotlin.` is unnecessary, I think just the plain `Suppress` is clear enough!
```suggestion
@Suppress("DSL_SCOPE_VIOLATION")
```